### ISSUE TITLE
Sparta Theme pagination minimum contrast adjustments

### ISSUE THREAD

**Issue by morganm58** [SAP/spartacus#15183]
The pagination is failing the minimum contrast adjustment. The font is too small and the font-weight needs to change as well.
